### PR TITLE
Khwahish

### PR DESCRIPTION
**Pull request by khwahish2oo2**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -105,5 +111,46 @@ def logout():
...
+# def redeem():
+#     employee=session['user_id']
+#     employee_points = User.query.filter_by(user_id=employee).first()
+#     if employee_points:
+#         points = employee_points.points
+#         return render_template('redeem.html', points=points)
+
+
+@app.route('/redeem_points', methods=['GET', 'POST'])
+def redeem_points():
```
Comment (MayankRawat06) [inline app.py:157]: make it just redeem

File: app.py [hunk 2]
```diff
@@ -62,6 +59,14 @@ def login():
 
     return render_template('login.html', title = 'Login', message=None)
 
+
+
+@app.route("/register", methods=['GET', 'POST'])
```
Comment (MayankRawat06) [inline app.py]: remove register section

File: static/styles.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+body {
```
Comment (MayankRawat06) [inline static/styles.css]: totlly different css, do something to integrate

File: templates/layout.html [hunk 4]
```diff
@@ -56,12 +56,17 @@
...
+                <a class="nav-link" href="/post_idea">post</a>
+              </li>
+            {% endif %}
             {% if session %}
               <li class="nav-item">
                 <a class="nav-link" href="/leaderboard">Leaderboard</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="">Redeem</a>
+                <a class="nav-link" href="/redeem_points">Redeem</a>
```
Comment (MayankRawat06) [inline templates/layout.html:64]: just redeem

File: templates/layout.html [hunk 5]
```diff
@@ -56,12 +56,17 @@
                 <a class="nav-link" href="/new_blog">Create</a>
               </li>
             {% endif %}
+            {% if session and session.get('is_manager', true) %}
+              <li class="nav-item">
+                <a class="nav-link" href="/post_idea">post</a>
```
Comment (MayankRawat06) [inline templates/layout.html]: why we're adding this?

**Review by MayankRawat06 — CHANGES_REQUESTED** [review]

File: templates/layout.html [hunk 6]
```diff
@@ -61,7 +61,10 @@
                 <a class="nav-link" href="/leaderboard">Leaderboard</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="">Redeem</a>
+                <a class="nav-link" href="/redeem_points">Redeem</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/Profile">Profile</a>
```
Comment (vijay-72) [inline templates/layout.html:67]: /profile would be more standard

**Pull request merged**